readit/database.py

```diff
@@ -109,6 +114,26 @@ def tag_url(self, tag_name, tagged_url):
         except Exception as t:
             print("Invalid input:--> ", t)
 
+    def list_all_tags(self):
+        """
+        Shows list of all available Tags in database.
+        """
+        tag_list = set()
+        try:
+            self.cursor.execute('''SELECT tags FROM bookmarks''')
+            tags_in_db = self.cursor.fetchall()
+            for tags_available in tags_in_db:
+                tag_list.add(tags_available)
+            tag_list = set(tag_list)
+            tag_list = list(tag_list)
```

Comment:
@ganeshhubale, if multiple URLs have the same `tag_name` then will `tag_list` return all the existing tags?